### ISSUE TITLE
usdview traceback when focusing in tree view with root prim selected

### ISSUE THREAD

**Issue by mds-dwa** [PixarAnimationStudios/USD#1619]
### Description of Issue
usdview prints a traceback if you press 'f' to focus in the tree view with the root prim selected, since it calls ```item = item.parent()``` before checking if item.parent() is valid. That results in the following error:

```
AttributeError: 'NoneType' object has no attribute 'parent'
```

A simple change to ExpandItemRecursively in [pxr/usdImaging/usdviewq/primTreeWidget.py](https://github.com/PixarAnimationStudios/USD/blob/release/pxr/usdImaging/usdviewq/primTreeWidget.py#L391) should resolve this:

```
    def ExpandItemRecursively(self, item):
        while item.parent():
            item = item.parent()
            if not item.isExpanded():
                self.expandItem(item)
```

### Steps to Reproduce
1. Open usdview
2. With mouse focus in the tree view and the 'root' prim still selected, press the 'f' key. Check the terminal for the (fortunately non-critical) traceback.

### System Information (OS, Hardware)
1. Linux, CentOS 7 (bug affects all platforms though)

### Package Versions
1. Tested bug and fix in 0.20.5, and relevant code doesn't seem to have changed and is still present in the latest builds.

### Build Flags
N/A (non-compiled python code change only)

**Comment by jilliene** [comment]
Filed as internal issue #USD-6897

**Comment by a1744633** [comment]
Hello @mds-dwa Just letting you know that we are working on this as software engineers at the University of Adelaide for a software project. Thanks for the issue!

**Comment by sunyab** [comment]
Closing, this was fixed by #1843 in the 22.08 release. Thanks!
